src/OpenTelemetry/README.md

```diff
@@ -379,19 +385,19 @@ A `FileSize`-KiB log file named as `ExecutableName.ProcessId.log` (e.g.
 `foobar.exe.12345.log`) will be generated at the specified directory
 `LogDirectory`, into which logs are written to.
 
-The SDK will attempt to open the configuration file in non-exclusive read-only
-mode, read the file and parse it as the configuration file every 10 seconds. If
-the SDK fails to parse the `LogDirectory`, `FileSize` or `LogLevel` fields as
+If the SDK fails to parse the `LogDirectory`, `FileSize` or `LogLevel` fields as
 the specified format, the configuration file will be treated as invalid and no
-log file would be generated. Otherwise, it will create or overwrite the log file
-as described above.
-
-Note that the `FileSize` has to be between 1 MiB and 128 MiB (inclusive), or it
-will be rounded to the closest upper or lower limit. When the `LogDirectory` or
-`FileSize` is found to be changed, the SDK will create or overwrite a file with
-new logs according to the new configuration. The configuration file has to be no
-more than 4 KiB. In case the file is larger than 4 KiB, only the first 4 KiB of
-content will be read.
+log file would be generated.
+
+When the `LogDirectory` or `FileSize` is found to be changed, the SDK will create
+or overwrite a file with new logs according to the new configuration. The
+configuration file has to be no more than 4 KiB. In case the file is larger than
+4 KiB, only the first 4 KiB of content will be read.
+
+The log file might not be a proper text file format to achieve the goal of having
+minimal overhead and bounded resource usage: it may have trailing `NULL`s if log
```

Comment:
Probably `NUL` instead of `NULL`? https://www.rapidtables.com/code/text/ascii-table.html